### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by hadih2pn**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
         <title>Welcome to Potionfy</title>
   </head>
   <body>
-    
+    <h1 style="color: red;">Welcome to Potionfy!</h1>
```
Comment (Hadihpn) [inline index.html:10]: set color blue

Comment (hadih2pn) [reply index.html:10]: sure can do

**Review by Hadihpn — CHANGES_REQUESTED** [review]
please do as my comment

**Review by Hadihpn — APPROVED** [review]
Well done  @hadih2pn

**Pull request merged**